openupgrade_scripts/scripts/account/17.0.1.2/upgrade_analysis_work.txt

```diff
@@ -0,0 +1,497 @@
+DEL account.tax.group: account.tax_group_taxes (noupdate)
+# DONE: removed in post-migration
+
+NEW ir.actions.act_window: account.action_account_all_payments
+NEW ir.actions.act_window: account.action_account_supplier_accounts
+NEW ir.actions.act_window: account.action_analytic_reporting
+NEW ir.actions.act_window: account.action_open_payment_items
+NEW ir.actions.act_window: account.action_open_sale_payment_items
+DEL ir.actions.act_window: account.action_account_tax_template_form
+DEL ir.actions.act_window: account.action_open_account_onboarding_create_invoice
+DEL ir.actions.act_window: account.action_open_account_onboarding_sale_tax
+NEW ir.actions.server: account.action_automatic_entry_change_account
+NEW ir.actions.server: account.action_automatic_entry_change_period
+NEW ir.actions.server: account.action_move_switch_move_type
+DEL ir.actions.server: account.action_automatic_entry
+DEL ir.actions.server: account.action_move_switch_invoice_to_credit_note
+NEW ir.cron: account.ir_cron_account_move_send
+NEW ir.model.access: account.access_account_journal_group_invoice
+NEW ir.model.access: account.access_account_journal_group_readonly
+NEW ir.model.access: account.access_account_move_send
+DEL ir.model.access: account.access_account_account_template
+DEL ir.model.access: account.access_account_chart_template
+DEL ir.model.access: account.access_account_fiscal_position_account_template
+DEL ir.model.access: account.access_account_fiscal_position_tax_template
+DEL ir.model.access: account.access_account_fiscal_position_template
+DEL ir.model.access: account.access_account_group_template
+DEL ir.model.access: account.access_account_invoice_send
+DEL ir.model.access: account.access_account_journal_group_all
+DEL ir.model.access: account.access_account_reconcile_model_line_template
+DEL ir.model.access: account.access_account_reconcile_model_template
+DEL ir.model.access: account.access_account_tag_internal_user
+DEL ir.model.access: account.access_account_tax_repartition_line_template_manager
+DEL ir.model.access: account.access_account_tax_template
+DEL ir.model.access: account.access_product_template_account_manager
+DEL ir.model.constraint: account.constraint_account_account_code_company_uniq
+DEL ir.model.constraint: account.constraint_account_fiscal_position_account_account_src_dest_uniq
+DEL ir.model.constraint: account.constraint_account_tax_name_company_uniq
+DEL ir.model.constraint: account.constraint_account_tax_template_name_company_uniq
+DEL ir.model.constraint: account_sequence.constraint_account_move_unique_name
+NEW ir.module.category: account.module_category_accounting_bank
+NEW ir.rule: account.account_move_send_rule_group_invoice (noupdate)
+NEW ir.rule: account.ir_rule_res_partner_bank_billing_officers (noupdate)
+NEW ir.rule: account.tax_group_comp_rule (noupdate)
+# NOTHING TO DO
+
+DEL ir.rule: account.account_invoice_send_rule_group_invoice (noupdate)
```

Comment:
you could have renamed it to `account_move_send_rule_group_invoice`, but it's ok.